Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -2324,4 +2345,29 @@ ErrorOr<Optional<ReadonlyBytes>> JPEGXLImageDecoderPlugin::icc_data()
         return OptionalNone {};
     return m_context->icc_profile();
 }
+
+}
+
+namespace AK {
+
+template<>
+struct Formatter<Gfx::Encoding> : Formatter<StringView> {
+    ErrorOr<void> format(FormatBuilder& builder, Gfx::Encoding const& header)
+    {
+        auto string = "Unknown"sv;
+        switch (header) {
+        case Gfx::Encoding::kVarDCT:
+            string = "VarDCT"sv;
```

Comment:
nit: if you `return Formatter<StringView>::format(builder, "VarDCT"sv);` here (and similar in the other branch, remove the `default`, and put a `VERIFY_NOT_REACHED()` after the switch, then clang will warn you on missing cases if you were to add a new entry to the enum. As-is, this here will then be silently outdated.

Reply:
Ah, I usually do this (at least I think). But here I just copied the formatter from somewhere else and didn't think twice about it, my bad.